src/core/forms.py

```diff
@@ -5,7 +5,7 @@
 
 from core import models, logic
 from submission import models as submission_models
-
+from setting_util import get_setting
```

Comment:
We like imports to be ordered (external to internal) and then alphabetically:

1. Python standard libraries
2. framework (Django)
3. libraries
4. internal libraries
5. modules
6. this module